fms/models/gpt_bigcode.py

```diff
@@ -0,0 +1,324 @@
+import math
+from dataclasses import dataclass
+from typing import Optional, Tuple
+
+import torch
+import torch.nn as nn
+from fms.utils.config import ModelConfig
+from fms.modules.feedforward import FeedForwardBlock
+from fms.utils.activation import str_to_activation
+
+from fms.modules.attention import MultiHeadAttention
+
+
+@dataclass
+class GPTBigCodeConfig(ModelConfig):
+    src_vocab_size: int = 49280
```

Comment:
What's the source for these various config values?

In HF I see vocab_size=50257 and in the original paper I see "vocabulary size of 49,152 tokens"

HF: https://huggingface.co/docs/transformers/model_doc/gpt_bigcode#transformers.GPTBigCodeConfig
paper: https://arxiv.org/pdf/2301.03988.pdf

Reply:
I believe I got this from some variation of hf santacoder. Having said that, I think it would make most sense to make it the same as the vocab_size from the paper.